tutorials/measurement_chain.ipynb

```diff
@@ -0,0 +1,553 @@
+{
```

Comment:
<p><span class="markdownDiffSpan" style="background: rgb(209, 255, 221) none repeat scroll 0% 0%; display: inline-block; --darkreader-inline-bgcolor:#114627; --darkreader-inline-bgimage:none;" data-darkreader-inline-bgcolor="" data-darkreader-inline-bgimage="">The only restriction is, that the function only has a single variable which represents the input signal. </span></p><p>In this example our input variable is given by <code>x</code>.</p><p> </p><p>The paragraph about the dimensionality is somehow confusing to people not familiar with pint, I'd say. Firstly say something about the actual or desired output dimensionality, e.g. temperature, voltage, current etc. and then tell what is the expected behaviour for handling these.</p><p>I guess you almost have that, but just clarify on <code>dimensionality</code>a  little bit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BAMWelDX/weldx/pull/326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Partially rewrote the section. Please have another look at it and tell me if it is clearer now</p>

Reply:
<p>It is clearer now, thank you. </p>